src/lib.rs

```diff
@@ -61,3 +63,33 @@ unsafe impl GlobalAlloc for LibcAlloc {
         libc::realloc(ptr as *mut c_void, new_size) as *mut u8
     }
 }
+
+#[cfg(target_family = "windows")]
+mod win_crt;
```

Comment:
Please keep module declarations bundled together at the top of the file (where the old `mod libc` declaration was)